projects/igniteui-angular/migrations/common/UpdateChanges.ts

```diff
@@ -221,15 +221,35 @@ export class UpdateChanges {
 
     protected updateClasses(entryPath: string) {
         let fileContent = this.host.read(entryPath).toString();
+        const alreadyReplaced = new Set<string>();
         for (const change of this.classChanges.changes) {
             if (fileContent.indexOf(change.name) !== -1) {
                 const positions = getRenamePositions(entryPath, change.name, this.service);
                 // loop backwards to preserve positions
                 for (let i = positions.length; i--;) {
                     const pos = positions[i];
-                    fileContent = fileContent.slice(0, pos.start) + change.replaceWith + fileContent.slice(pos.end);
+                    // V.S. 18th May 2021: If several classes are renamed w/ the same import, erase them
+                    if (i === 0 && alreadyReplaced.has(change.replaceWith)) {
+                        const trailingCommaWhiteSpace = new RegExp(/,([\s]*)(?=(\s}))/, 'g');
```

Comment:
This can probably be improved by using TS compiler API/LS instead of somewhat clunky string manipulation. Either way, guess it'll do for now. 
Just one question, about this regex - are you sure you want to keep it global, considering this is run on the entire content? Some formatting styles leave trailing comma on purpose and this can potentially replace parts unrelated to the migration. Perhaps limit it to one replace at the start of `afterReplace` instead?